include/aikido/robot/util.hpp

```diff
@@ -200,6 +200,32 @@ trajectory::TrajectoryPtr planToEndEffectorOffset(
     = VectorFieldPlannerParameters(),
     const CRRTPlannerParameters& crrtParameters = CRRTPlannerParameters());
 
+/// Plan with a desired end-effector twist.
+/// \param[in] space StateSpace for the metaskeleton
+/// \param[in] metaSkeleton Metaskeleton to plan with
+/// \param[in] body Bodynode for the end-effector
+/// \param[in] twistSeq Twist for the end-effector
+/// \param[in] durationSeq Time to plan with the desired twist
```

Comment:
What is the reasoning behind naming `twistSeq` and `durationSeq`? Why not `twist` and `timeout`?